mypyc/test-data/run-lists.test

```diff
@@ -103,6 +103,12 @@ pop(l, -2)
 assert l == [1, 3]
 assert count(l, 1) == 1
 assert count(l, 2) == 0
+l.insert(0, 0)
+assert l == [0, 1, 3]
+l.insert(2, 2)
+assert l == [0, 1, 2, 3]
+l.insert(4, 4)
+assert l == [0, 1, 2, 3, 4]
```

Comment:
@vsakkas This doesn't test the new primitive, since the code is in `driver.py`, which doesn't get compiled. Would you mind updating the test case? Also, it would be good to check negative indexes and index underflow and overflow.

Reply:
Of course. I'll look into updating the tests in a new PR.